pyctcdecode/tests/test_decoder.py

```diff
@@ -600,3 +600,29 @@ def test_serialization(self):
 
         # this decoder has a language model so we expect one more key
         self.assertEqual(old_num_models + 1, self._count_num_language_models())
+
+    def test_load_from_hub_offline(self):
```

Comment:
In order for this test to work this PR has to be merged: https://github.com/huggingface/huggingface_hub/pull/505 . This should only take a day or so though